guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc

```diff
@@ -121,7 +121,7 @@ For distributing content from custom sources you need to create products and rep
 Red{nbsp}Hat repositories are signed with GPG keys by default, and it is recommended to create GPG keys also for your custom repositories.
 The configuration of custom repositories depends on the type of content they hold (RPM packages, or Docker images).
 
-Repositories configured as `yum` repositories, that contain only RPM packages, can make use of the new download policy setting to save on synchronization time and storage space.
+Yum repositories that contain only RPM packages can make use of the new download policy setting to save on synchronization time and storage space.
```

Comment:
"new download policy setting" is a bit vague. Also, anything that is "new" will soon be "old". :-)

Perhaps change to "Yum repositories that contain only RPM packages support the **On demand** policy setting, which reduces synchronization time and storage space."

Reply:
I have reworded this as suggested.